parquet-column/src/main/java/org/apache/parquet/io/api/Binary.java

```diff
@@ -542,6 +606,10 @@ public static Binary fromCharSequence(CharSequence value) {
     return new FromCharSequenceBinary(value);
   }
 
+  public static int lexicographicCompareTo(Binary one, Binary other) {
```

Comment:
Still contains `To`.
```suggestion
  public static int lexicographicCompare(Binary one, Binary other) {
```